packages/build/src/plugins_core/secrets_scanning/utils.ts

```diff
@@ -118,10 +116,10 @@ export async function getFilePathsToScan({ env, base }): Promise<string[]> {
   // this is needed for windows machines and snapshot tests consistency.
   files = files.map((f) => f.split(path.sep).join('/'))
 
-  let omitPaths: string[] = []
+  // always omit .git directories as they can be massive and are not the current version of the code or build output
+  let omitPaths: string[] = ['.git/']
   if (typeof env.SECRETS_SCAN_OMIT_PATHS === 'string') {
     omitPaths = env.SECRETS_SCAN_OMIT_PATHS.split(',')
```

Comment:
This way, `.git/` might still not get ignored if `SECRETS_SCAN_OMIT_PATHS` is supplied and `.git/` isn't part of it. Just checking that's intended vs. always forcing `.git/` to be ignored?

Reply:
actually, that's not the intent. That's for looking closely there. I need to avoid stress coding 😅